presto-docs/src/main/sphinx/functions/array.rst

```diff
@@ -258,6 +258,17 @@ Array Functions
         SELECT array_top_n(ARRAY [1, 100], 5); -- [100, 1]
         SELECT array_top_n(ARRAY ['a', 'zzz', 'zz', 'b', 'g', 'f'], 3); -- ['zzz', 'zz', 'g']
 
+.. function:: array_transpose(array(array(T))) -> array(array(T))
```

Comment:
I believe this is from a plugin which is not loaded by default in Presto clusters.  We should probably add this documentation elsewhere.